### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by haris741**
Added the method to clone the repository by cloning link directly in the android studio without downloading the zip file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,8 @@ Before you begin, you should already have the Android Studio SDK downloaded and
 
 ### Setting up the Android Project
 
+### By downloading the zip file
```
Comment (CloudyPadmal) [inline README.md]:
```suggestion
### Downloading the zip file
```

File: README.md [hunk 2]
```diff
@@ -159,6 +161,40 @@ To debug over Wi-Fi: http://blog.fossasia.org/android-app-debugging-over-wifi-fo
 Note :
 1. If you built your own hardware, change VendorID and/or ProductID in [CommunicationHandler.java](https://github.com/fossasia/pslab-android/blob/master/app/src/main/java/io/pslab/communication/CommunicationHandler.java)
 
+### By direct cloning
```
Comment (CloudyPadmal) [inline README.md]:
```suggestion
### Cloning
```

File: README.md [hunk 3]
```diff
@@ -159,6 +161,40 @@ To debug over Wi-Fi: http://blog.fossasia.org/android-app-debugging-over-wifi-fo
...
+
+
```
Comment (CloudyPadmal) [inline README.md]: Remove unwanted line breaks

File: README.md [hunk 4]
```diff
@@ -159,6 +161,40 @@ To debug over Wi-Fi: http://blog.fossasia.org/android-app-debugging-over-wifi-fo
...
+1. Open Android Studio, you will see a **Welcome to Android** window. Under Quick Start, select "check out project from version control".
```
Comment (CloudyPadmal) [inline README.md]: Use similar formatting as in  _Import Project (Eclipse ADT, Gradle, etc.)_

File: README.md [hunk 5]
```diff
@@ -159,6 +161,40 @@ To debug over Wi-Fi: http://blog.fossasia.org/android-app-debugging-over-wifi-fo
...
+
+2. Select git from the drop down menu that appeared.
+
+3. Go to the repository and click clone or download button.
```
Comment (CloudyPadmal) [inline README.md]: Emphasize these commands/buttons

**Comment by CloudyPadmal** [comment]
Hello @haris741 , are you still working on this PR? There seems to be a new one with same content in #2075

**Comment by mariobehling** [comment]
Closing this due to inactivity and overlap with another PR that was more active.

**Pull request closed without merging**